Sergey_Shashin/src/components/Messanger/Messanger.jsx

```diff
@@ -8,39 +8,116 @@ import { Header } from 'components/Header/Header';
 
 export class Messanger extends Component {
   state = {
-    messages: [],
+    chats: {
+      '1': {
+        id: 1,
+        messages: [
+          {
+            text: 'Привет! Это чат №1',
+            author: 'Bot',
+          },
+        ],
+        name: 'Chat 1',
+      },
+      '2': {
+        id: 2,
+        messages: [
+          {
+            text: 'Привет! Это чат №2',
+            author: 'Bot',
+          },
+        ],
+        name: 'Chat 2',
+      },
+      '3': {
+        id: 3,
+        messages: [
+          {
+            text: 'Привет! Это чат №3',
+            author: 'Bot',
+          },
+        ],
+        name: 'Chat 3',
+      },
+      '4': {
+        id: 4,
+        messages: [
+          {
+            text: 'Привет! Это чат №4',
+            author: 'Bot',
+          },
+        ],
+        name: 'Chat 4',
+      },
+      '5': {
+        id: 5,
+        messages: [
+          {
+            text: 'Привет! Это чат №5',
+            author: 'Bot',
+          },
+        ],
+        name: 'Chat 5',
+      },
+    }
   }
 
   componentDidUpdate() {
-    const lastMessage = this.state.messages[this.state.messages.length - 1];
+    if (this.messages.length) {
+      const lastMessage = this.messages[this.messages.length - 1];
+
+      if (lastMessage.author != 'Bot') {

```

Comment:
Проверку лучше осуществлять без приведения типа
```suggestion
      if (lastMessage.author !== 'Bot') {
```